monitoring/monitorlib/scd_automated_testing/scd_injection_api.py

```diff
@@ -111,7 +111,6 @@ class ClearAreaOutcome(ImplicitDict):
 
 
 class ClearAreaResponse(ImplicitDict):
-    request: ClearAreaRequest
     outcome: ClearAreaOutcome
```

Comment:
If this were changed to:

```python
class ClearAreaResponse(ImplicitDict):
    request: Optional[ClearAreaRequest]
    outcome: ClearAreaOutcome
```

...then this PR would look good to me.

Reply:
Will do.